src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -235,6 +235,33 @@ public function getEnabledTransitions(object $subject)
         return $enabledTransitions;
     }
 
+    public function getEnabledTransition(object $subject, string $name, array $froms = null, array $tos = null): ?Transition
+    {
+        $marking = $this->getMarking($subject);
+
+        $result = array_filter(
```

Comment:
I guess you were right, this is not really possible to have many transitions with the same name enabled at the very same time.  
I'm sorry for loosing your time, but could you revert this change to the one you had it yesterday? the previous version was more readable :)

Thanks

Reply:
done !